apk-generator/v2/app/generator/generator.py

```diff
@@ -327,24 +329,37 @@ def run_command(self, command):
         rc = process.poll()
         return rc
 
+    @socketio.on('message')
```

Comment:
this helps me to get reference in the js code. thats why used a decorator. am i wrong?